en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,146 @@
+
+* `/read `: It will first check if the file exists. If so it will read the content of the file. 
+<p><img src="../../../assets/img/connectors/FileConnector-02.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+## Configure the connector in WSO2 Integration Studio
+
+1. Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+[Configure the Connector and the Connector Exporter Project](../importing-connector-to-integration-studio.md).
+
+2. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API. 
+<p><img src="../../../assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/></p>
+
+3. Provide the API name as File Connector and the API context as `/fileconnector`. You can go to the source view of the xml configuration file of the API and copy the following configuration. 
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/fileconnector" name="FileConnector" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="POST" uri-template="/create">
+        <inSequence>
+            <property expression="json-eval($.source)" name="source" scope="default" type="STRING"/>
+            <property expression="json-eval($.inputContent)" name="inputContent" scope="default" type="STRING"/>
+            <fileconnector.create>
+                <source>{$ctx:source}</source>
+                <inputContent>{$ctx:inputContent}</inputContent>
+            </fileconnector.create>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+    <resource methods="POST" uri-template="/read">
+        <inSequence>
+            <property expression="json-eval($.source)" name="source" scope="default" type="STRING"/>
+            <fileconnector.isFileExist>
+                <source>{$ctx:source}</source>
+            </fileconnector.isFileExist>
+            <property expression="json-eval($.fileExist)" name="response" scope="default" type="STRING"/>
+            <log level="custom">
+                <property expression="get-property('response')" name="responselog"/>
+            </log>
+            <switch source="get-property('response')">
+                <case regex="true">
+                    <fileconnector.read>
+                        <source>{$ctx:source}</source>
+                    </fileconnector.read>
+                    <respond/>
+                </case>
+                <default>
+                    <log>
+                        <property name="notext" value="&quot;File does not exist&quot;"/>
+                    </log>
+                </default>
+            </switch>
+            <drop/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </resource>
+</api>
+
+```
+
+4. Follow these steps to [export the artifacts](../exporting-artifacts.md).
```

Comment:
Here as well we need to check. @samgnaniah

Reply:
This will not work. Has to be in the following format:

{! ../../file-name.md !}